spec/integration/client/client_spec.rb

```diff
@@ -72,13 +72,18 @@ def remove_registry_key
 
   def verify_export_password_exists
     powershell_exec! <<~EOH
-      Try {
-          $response = Get-ItemPropertyValue -Path "HKLM:\\Software\\Progress\\Authentication" -Name "PfxPass" -ErrorAction Stop
-          if ($response) {return $true}
-      }
-      Catch {
-          return $false
+    Try {
+      if (-not (($PSVersionTable.PSVersion.Major -ge 5) -and ($PSVersionTable.PSVersion.Build -ge 22000)) ) {
+        $response = Get-ItemProperty -Path "HKLM:\\Software\\Progress\\Authentication" -Name "PfxPass" -ErrorAction Stop
+        }
```

Comment:
Would this work on old AND new? If so, should we get rid of the conditional logic and throw in a comment as to why we are using it over `Get-ItemPropertyValue`? I think that would make this cleaner.

Reply:
Oh gotcha. Good Point